v22.1/demo-fault-tolerance-and-recovery.md

```diff
@@ -291,48 +283,41 @@ To be able to tolerate 2 of 5 nodes failing simultaneously without any service i
     --background
     ~~~
 
-2. Use the [`ALTER RANGE ... CONFIGURE ZONE`](configure-zone.html) command to change the cluster's `default` replication factor to 5:
+1. Use the [`ALTER RANGE ... CONFIGURE ZONE`](configure-zone.html) command to change the cluster's `default` replication factor to 5:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ cockroach sql --execute="ALTER RANGE default CONFIGURE ZONE USING num_replicas=5;" --insecure --host=localhost:26000
+    $ cockroach sql --insecure --host=localhost:26000 \
+    --execute="ALTER RANGE default CONFIGURE ZONE USING num_replicas=5;"
     ~~~
 
-3. Back in the DB Console **Overview** dashboard, watch the replica count increases and even out across all 6 nodes:
+1. In the DB Console **Overview** dashboard, watch the replica count increase and even out across all 6 nodes:
 
-    <img src="{{ 'images/v22.1/fault-tolerance-8.png' | relative_url }}" alt="DB Console" style="border:1px solid #eee;max-width:100%" />
+    <img src="{{ 'images/v22.1/fault-tolerance-8.png' | relative_url }}" alt="DB Console Cluster Restore" style="border:1px solid #eee;max-width:100%" />
 
     This shows the cluster up-replicating so that each range has 5 replicas, one on each node.
 
 ## Step 9. Simulate two simultaneous node failures
 
-Gracefully shut down two nodes, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure):
-
-{% include_cached copy-clipboard.html %}
-~~~ shell
-kill -TERM 4510
-~~~
+Gracefully shut down **2 nodes**, specifying the [process IDs you retrieved earlier](#step-5-simulate-a-single-node-failure):
 
 {% include_cached copy-clipboard.html %}
 ~~~ shell
-kill -TERM 4622
+kill -TERM {process-id}
```

Comment:
This change makes it look as though only one pid is referenced. We should specify exact process IDs from the example, for clarity:

```
{% include_cached copy-clipboard.html %}
~~~ shell
kill -TERM 53708 53730
~~~
```

Reply:
I can't specify `53708` here because that process no longer exists since it was killed in a preceding step. That's why I give the generic kill example and say to shut down 2 nodes.  When a user runs this tutorial the process IDs will be different anyway.

Reply:
That node is restarted in Step 8. I assumed that it would have the same process ID, but I can't recall what happened when I tested these steps. 

But your latter point is taken. In that case, I think the command should read

```
{% include_cached copy-clipboard.html %}
~~~ shell
kill -TERM {process IDs}
~~~
```

Reply:
It doesn't have the same process ID. I think this solution is much better. Thanks!